aws/aws-how-to/instances/automatically-setup-multiple-nics.rst

```diff
@@ -0,0 +1,55 @@
+Set up automated network configuration for instances with multiple NICs
+=======================================================================
+
+Ubuntu instances running on AWS can be configured to automatically configure source routing on multiple NICs via cloud-init, starting with `v24.1`_, on different `events`_, as:
+
+* First boot (enabled by default)
+* Subsequent boots
+* Hotplug NIC additions and removals
+
+.. note::
+
+   On Ubuntu Noble and newer, hot-plugged NICs are configured by default.
+
+Enable additional network configuration events
+----------------------------------------------
+
+To enable network configuration on additional `events`_, one can add supplemental user-data at instance creation time:
+
+..  code-block:: yaml
+
+    #cloud-config
+    # apply network config on every hotplug event
+    updates:
+      network:
+        when: ['hotplug']
+
+..  code-block:: yaml
+
+    #cloud-config
+    # apply network config on every boot and hotplug event
+    updates:
+      network:
+        when: ['boot', 'hotplug']
+
```

Comment:
Instead of the reference section at the end, maybe we could add a statement here, something like: 
"To learn more about configuring instances using cloud-init, refer to `cloud-init datasources`_ (with a link to "https://docs.cloud-init.io/en/latest/reference/datasources/ec2.html") and `EC2 as a cloud-init datasource`_ (with a link to "https://docs.cloud-init.io/en/latest/reference/datasources/ec2.html").